tslib/src/functions/ps-report-transformer/report-line.class.ts

```diff
@@ -0,0 +1,95 @@
+import {
+  AnswersOrNull,
+  CheckConfigOrNull,
+  CheckFormOrNull,
+  CheckOrNull, DeviceOrNull, EventsOrNull,
+  Pupil,
+  School
+} from '../../functions-throttled/ps-report-2-pupil-data/models'
+import { PsychometricReportLine } from './models'
+
+export class ReportLine {
+  private readonly _answers: AnswersOrNull
+  private readonly _check: CheckOrNull
+  private readonly _checkConfig: CheckConfigOrNull
+  private readonly _checkForm: CheckFormOrNull
+  private readonly _device: DeviceOrNull
+  private readonly _events: EventsOrNull
+  private readonly _pupil: Pupil
+  private readonly _school: School
+  private _report: PsychometricReportLine = {
+    // Pupil fields
+    DOB: '',
+    Gender: '',
+    PupilID: '',
+    Forename: '',
+    Surname: '',
+    ReasonNotTakingCheck: null
+  }
+
+  constructor (
+    answers: AnswersOrNull,
```

Comment:
if i recall correctly there is a shorthand way of doing constructor params with backing fields.
not always ideal, but worth knowing

Reply:
Yes, I am aware.  But here we are using getters and setters so the `answers` param is being set to the `_answers` property.